src/pages/Autolending/AutolendingPage.vue

```diff
@@ -10,6 +10,7 @@
 			<main-toggle />
 		</div>
 		<div class="row column settings-area" :class="{ obscure: !isEnabled }">
+			<lend-timing-dropdown v-if="isEnabled" />
```

Comment:
The v-if should not be needed as this dropdown will always be visible.

Reply:
I think it looks better if the drop down is hidden because the copy reads: 

_Auto-lending On
Kiva will automatically lend my balance
<dropdown>_

and 

_Auto-lending off
I’ll lend my balance myself_

If the dropdown was not hidden it would be: 

_Auto-lending off
I’ll lend my balance myself
<dropdown>_

Reply:
The design mocks have it still visible, but transparent and non-interactive. See the timing-donation-OFF.png attachment in https://kiva.atlassian.net/browse/CASH-1249